From 81bb6daebef02481f6bc467a64d1402c598627dd Mon Sep 17 00:00:00 2001
From: undef <gitlab@undef.tools>
Date: Tue, 21 Sep 2021 08:56:22 +0000
Subject: [PATCH 4/8] Wrap long labels on small screens

Both the keyboard and region panels contain long GtkLabels. These labels
cause gnome-control-center overflow the screen, especially on small
screens like those on phones.

From https://gitlab.gnome.org/GNOME/gnome-control-center/-/merge_requests/1057
---
 panels/keyboard/cc-keyboard-panel.ui | 2 ++
 panels/region/cc-region-panel.ui     | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/panels/keyboard/cc-keyboard-panel.ui b/panels/keyboard/cc-keyboard-panel.ui
index 9b0952e88..b8e8194d9 100644
--- a/panels/keyboard/cc-keyboard-panel.ui
+++ b/panels/keyboard/cc-keyboard-panel.ui
@@ -128,6 +128,7 @@
                                 <property name="visible">True</property>
                                 <property name="margin_left">6</property>
                                 <property name="use_underline">True</property>
+                                <property name="wrap">True</property>
                                 <property name="label" translatable="yes">Use the _same source for all windows</property>
                               </object>
                             </child>
@@ -155,6 +156,7 @@
                                 <property name="visible">True</property>
                                 <property name="margin_left">6</property>
                                 <property name="use_underline">True</property>
+                                <property name="wrap">True</property>
                                 <property name="label" translatable="yes">Switch input sources _individually for each window</property>
                               </object>
                             </child>
diff --git a/panels/region/cc-region-panel.ui b/panels/region/cc-region-panel.ui
index fc698f0af..d3a4801fb 100644
--- a/panels/region/cc-region-panel.ui
+++ b/panels/region/cc-region-panel.ui
@@ -86,6 +86,7 @@
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
                     <property name="xalign">0</property>
+                    <property name="wrap">True</property>
                     <property name="label" translatable="yes">The language used for text in windows and web pages.</property>
                     <style>
                       <class name="dim-label"/>
@@ -211,6 +212,7 @@
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
                     <property name="xalign">0</property>
+                    <property name="wrap">True</property>
                     <property name="label" translatable="yes">The format used for numbers, dates, and currencies.</property>
                     <style>
                       <class name="dim-label"/>
-- 
2.25.1

